src/containers/shared/components/Transaction/AMM/AMMCreate/Simple.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { SimpleRow } from '../../SimpleRow'
+import { TransactionSimpleProps } from '../../types'
+import { formatTradingFee } from '../../../../utils'
+import { Account } from '../../../Account'
+import { Amount } from '../../../Amount'
+
+export const Simple = ({ data }: TransactionSimpleProps) => {
+  const { t } = useTranslation()
+  const { amount, amount2, accountID, fee, tradingFee } = data.instructions
```

Comment:
`fee` is no longer in the parser.

Reply:
I removed these from ammCreate since we use the amounts from the transaction instead of the metadata - these are still necessary for other transactions that get amounts from the metadata.